src/sql/base/browser/ui/infoBox/infoBox.ts

```diff
@@ -43,10 +52,26 @@ export class InfoBox extends Disposable implements IThemable {
 		container.appendChild(this._infoBoxElement);
 		this._infoBoxElement.appendChild(this._imageElement);
 		this._infoBoxElement.appendChild(this._textElement);
+
+		this._clickableIndicator = DOM.$('.infobox-clickable-arrow');
```

Comment:
this is not keyboard accessible. please use button instead. and remember to set aria-label for button.

Reply:
@aasimkhan30 Not sure if you were going to be doing more here - but you should be creating a new Button object and using that instead of creating a div element and making it act like a button.

Reply:
Actually, let's first start with how is the Azure portal doing it? Button element or div that acts like a button?